mlir/lib/Catalyst/Transforms/BufferizationPatterns.cpp

```diff
@@ -82,13 +82,26 @@ struct BufferizeCustomCallOp : public OpConversionPattern<CustomCallOp> {
     }
 };
 
+struct BufferizePythonCallOp : public OpConversionPattern<PythonCallOp> {
+    using OpConversionPattern::OpConversionPattern;
+
+    LogicalResult matchAndRewrite(PythonCallOp op, OpAdaptor adaptor,
+                                  ConversionPatternRewriter &rewriter) const override
+    {
+        rewriter.create<PythonCallOp>(op.getLoc(), adaptor.getOperands(), adaptor.getIdentifier());
```

Comment:
Is it necessary, as it seems to me that it does not modify the operation? (not sure if the adaptor has different types maybe)

Reply:
The adaptor has different types. It changes from tensors to memrefs.